src/pages/hotel/search/components/room-availability/rooms/RoomList.tsx

```diff
@@ -117,7 +119,7 @@ function RoomList({
 
                     <div className="flex ">
                       <IoIosCheckmark className="text-lg text-green-600 ml-2 mt-2"></IoIosCheckmark>
-                      {room.isCancellable ? (
+                      {room.iscancellable ? (
```

Comment:
revert to `isCancellable` it self w.r.t above comment

Reply:
same as above